spec/rubocop/cop/ezcater/feature_flag_active_spec.rb

```diff
@@ -11,7 +11,7 @@
 
   before { inspect_source(line) }
 
-  %w(EzcaterFeatureFlag EzFF).each do |constant_name|
+  %w(EzcaterFeatureFlag EzFF ::EzFF ::EzcaterFeatureFlag).each do |constant_name|
```

Comment:
It doesn't look like we needed to do anything to fix namespacing in this cop, but I appreciate adding additional tests when they're ones like this that probably execute very quickly.